docs/kyma-environment-broker/03-17-runtime-reconciler.md

```diff
@@ -0,0 +1,31 @@
+# Runtime reconciler
+
+Runtime reconciler is application which perform some reconcile tasks on runtimes (SKRs).
+
+## Overview
+
+Currently there is one task for runtime reconciler, it is to reconcile BTP Manager secrets on SKRs. It is achived in two ways, first one is implemention with usage of runtime-watcher, which in real time sends event about changes of secret from SKR to KEB. The second method is a job, which periodically loop over all instances from KEB database which have existing assigned runtimes ID, and for each of them, do a check if secret on SKR match with credentials from KEB database.
```

Comment:
```suggestion
Currently, there is one task for Runtime Reconciler. It reconciles BTP Manager Secrets on SAP BTP, Kyma runtimes (SKRs). It can do it in two ways: 
- by implementation with the usage of Runtime Watcher, which sends events about changes of the Secret from the SKR to KEB in real time. 
- with a job, which periodically loops over all instances from the KEB database; each instance has an existing assigned runtime ID; the job checks if the Secret on the SKR matches the credentials from the KEB database.
```